docs/_main/getting_started.md

```diff
@@ -22,23 +22,19 @@ status. Before we can call any methods, we need to configure our new app with th
 ## Getting a token to use the Web API
```

Comment:
I know you didn't touch the previous section but while we're touching this file could change `register a new app` -> `create a new app`